src/index.ts

```diff
@@ -18,14 +20,38 @@ export const run = () => {
       }
     });
 
-    if (!args[0] || !AVAILABLE_COMMANDS.includes(args[0])) {
+    if (!args[0]) {
       showHelp(args[0], options.help);
-    } else if (args[0] === 'android') {
-      const androidSetup = new AndroidSetup(options);
-      androidSetup.run();
     } else {
-      const iOSSetup = new IosSetup(options);
-      iOSSetup.run();
+      if (args[0].includes('.')) {
+        // Handle dot commands
+        const dotCommandArgs = args[0].split('.');
+        if (dotCommandArgs.length !== 2 || !AVAILABLE_DOT_COMMANDS.includes(dotCommandArgs[0])) {
+          showHelp(args[0], options.help);
+        } else {
+          if (dotCommandArgs[0] === 'android') {
+            if (!ANDROID_DOT_COMMANDS.includes(dotCommandArgs[1])) {
+              showHelp(args[0], options.help);
+            } else {
+              const androidDotCommand = new AndroidDotCommand((dotCommandArgs[1] as SdkBinary), args[1]);
+              androidDotCommand.run();
+            }
+          }
+        }
```

Comment:
We don't need to handle the `.` commands with any separate logic. In the existing logic that we just merged, replace the `args[0]` with `args[0].split('.')[0]` everywhere and things should work the same. Then, inside the else if block where we check for 'android', take the complete flow to a separate `commands/android/index.ts` file and handle everything there.

Reply:
Also, instead of actually replacing `args[0]` everywhere like above, take the new value into a variable called `command` or `cmd` and use it everywhere.